Makefile

```diff
@@ -0,0 +1,19 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+ifeq ($(WITH_COVERAGE), TRUE)
+	TEST_COMMAND = coverage run -m skbio/__init__
```

Comment:
That's odd. I went ahead and changed it. Either way works in m bash 4.3. Are you on Mac?